RetrospectiveExtension.Frontend/components/feedbackItemGroup.tsx

```diff
@@ -49,7 +51,7 @@ export default class FeedbackItemGroup extends React.Component<IFeedbackItemGrou
     // Bug 19016440: Edge drag and drop dataTransfer protocol is bugged
     // const droppedItemId = e.dataTransfer.getData('id');
     const droppedItemId = localStorageHelper.getIdValue();
-    FeedbackItem.handleDropFeedbackItemOnFeedbackItem(this.props.mainFeedbackItem, droppedItemId, this.props.mainFeedbackItem.id);
+    FeedbackItemStatic.handleDropFeedbackItemOnFeedbackItem(this.props.mainFeedbackItem, droppedItemId, this.props.mainFeedbackItem.id);
```

Comment:
same with the previous comment

_this time for `handleDropFeedbackItemOnFeedbackItem` method_